data_decl.go

```diff
@@ -67,6 +67,7 @@ type Params struct {
 	httpClientTimeout     int
 	connectTimeout        int
 	TLSHandshakeTimeout   int
+	insecureSkipVerify    bool
```

Comment:
I suggest to rename for better clarity.  E.g. `skipSSLCertVerification`.

Reply:
Makes sense! I had gone with the aws-sdk-go naming convention but like you said, this will make it more apparent. I'll make the necessary changes.

Reply:
@t7ko-seagate I have renamed the parameter as suggested. Please have a look and let me know if any further changes/corrections are needed.